src/util/prolog/prologResult.ts

```diff
@@ -54,7 +54,7 @@ export class PrologResult {
 
 	/**
 		* Alternative way to access the results via an array.
-		* @return {Array<Array<string>>} First Level: i-1-th answer Second Level: Variable in query's order
+		* @return {Array<Array<string>>} First Level: i-th answer set Second Level: Variable in query's order
```

Comment:
Nit:
What do you mean by i-th answer **set**?
Do you really mean the verb "set"? If so: When is the answer set or by whom?

Or do you mean the noun "set", referring to the mathematical notion of a collection of elements? If so: Does your datatype `Array<Array<string>>` really have the properties of a set?

Reply:
There is a mathematical set which follows more strict rules, but here I don't mean the mathematical definition. I wouldn't know a better description for this though I was thinking the same.